src/instantsearch/instantsearch.ts

```diff
@@ -299,16 +250,8 @@ export class NgAisInstantSearch implements AfterViewInit, OnInit, OnDestroy {
   }
 
   public createInstantSearchInstance(config: InstantSearchConfig) {
-    // add default searchParameters with highlighting config
-    if (!config.searchParameters) config.searchParameters = {};
-    Object.assign(config.searchParameters, {
```

Comment:
this is default in InstantSearch now IIRC

Reply:
yes, since: https://github.com/algolia/instantsearch.js/pull/4001/files#diff-138bea1bb7b3aaf1d85f9a648c73ea26R139